reference/curl/curlstringfile/construct.xml

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="utf-8"?>
+<refentry xml:id="curlstringfile.construct" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+ <refnamediv>
+  <refname>CURLStringFile::__construct</refname>
+  <refpurpose>Erstellt ein CURLStringFile-Objekt</refpurpose>
+ </refnamediv>
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <constructorsynopsis role="CURLStringFile">
+   <modifier>public</modifier> <methodname>CURLStringFile::__construct</methodname>
+   <methodparam><type>string</type><parameter>data</parameter></methodparam>
+   <methodparam><type>string</type><parameter>postname</parameter></methodparam>
+   <methodparam choice="opt"><type>string</type><parameter>mime</parameter><initializer>"application/octet-stream"</initializer></methodparam>
+  </constructorsynopsis>
+  <para>
+   Erstellt ein <classname>CURLStringFile</classname>, welches zum Upload einer
```

Comment:
I'd suggest <classname>CURLStringFile</classname>-Objekt, ...

Reply:
Definitely makes sense. I will adjust that in a fixup commit later (to keep the commit history clean).